test/appSearch.test.js

```diff
@@ -0,0 +1,112 @@
+const {getBearerToken} = require('../src/app/auth')
+const faker = require('faker')
+const sinon = require("sinon")
+const axios = require('axios')
+
+let req
+let resSpy = {}
+let authToken
+let nextSpy
+let serverUrl = 'http://localhost:3000/app'
```

Comment:
Try splitting the paramaters that are re-assinged during the tests, then the consts ones.
req,headers,serverUrl can be changed to const. It will help us to tell which params are changing, and prevent mistakes of reassigning them.

Also, seems like 'nextSpy' isn't necessary, is it?